server/index.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../dist/static/js/bundle.js'));
+
+const PORT = 3101;
+
+app.get('/tiersIncluded', (req, res) => {
+  const dataString = { bundleId: 1 };
```

Comment:
I think it may be easier to follow if you just called the incoming argument bundleId and kept it so (or as some iteration throughout). 

How will dataString receive the value of the bundle id? I would indicate if it is using request query ( `const dataString = req.query.`_whatever the name of your query variable here_ ) or request params ( ...(`'/tiersIncluded/:bundleId`)... `const dataString = req.params.bundleId;` ). Then keep this line for when you need it but keep it commented out until then.

Reply:
This particular method is only for testing and thus has a hardcoded value. Adding a comment for clarity.